spec/controllers/customer_orders_controller_spec.rb

```diff
@@ -23,7 +23,7 @@
     specify do
       expect(response).to have_http_status(200)
       expect(response).to render_template(:index)
-      expect(assigns(:customer_orders)).to eq [customer_order_1, customer_order_2]
+      expect(assigns(:customer_orders)) == [customer_order_1, customer_order_2]
```

Comment:
sao lại dùng `==` mà lại ko dùng như thế này?
`expect(assigns(:customer_orders)).to eq [customer_order_1, customer_order_2]`

Reply:
expect(assigns(:customer_orders).to_a).to eq [customer_order_1, customer_order_2]